src/Elewant/AppBundle/Entity/CustomType/Breed.php

```diff
@@ -33,6 +33,9 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
      */
     public function convertToPHPValue($value, AbstractPlatform $platform)
     {
+        if ($value === null) {
```

Comment:
Can I ask why this was this needed? I think if we ever have null values as Breed in a projection, we've got problems in our projection...